nixos/modules/services/web-apps/crater.nix

```diff
@@ -0,0 +1,121 @@
+{ config, lib, pkgs, ... }:
+
+let
+  inherit (lib) mkDefault mkEnableOption mkIf mkOption types;
+
+  cfg = config.services.crater;
+  fpm = config.services.phpfpm.pools.crater;
+  package = pkgs.crater.override { dataDir = cfg.dataDir; };
+in
+{
+  options.services.crater = {
+
+    enable = mkEnableOption (lib.mdDoc "Crater");
```

Comment:
You should probably mention that the user still has to perform an install via the web ui after they enable this.